app/AppKernel.php

```diff
@@ -37,11 +37,15 @@ public function registerBundles()
             new FOS\RestBundle\FOSRestBundle(),
             new JMS\SerializerBundle\JMSSerializerBundle(),
 
+            // Application bundles
+            new AppBundle\AppBundle(),
+
             // Remove this Bundle when using the SE as the basis for a new project
             new Acme\DemoBundle\AcmeDemoBundle(),
```

Comment:
That's because the DemoBundle isn't installed by default anymore, only when you answer "y" to the aswer "Do you want to install the demo bundle?". This is then handled by the SensioDistributionBundle, where the demo bundle lives in Resources/skeleton.